modules/monitoring-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-limits-and-resource-requests-for-monitoring-components_{context}"]
+= Specifying limits and requests for monitoring components 
+
+To configure CPU and memory resources, specify values for resource limits and requests in the appropriate `ConfigMap` object for the namespace in which the monitoring component is located:
+
+* The `cluster-monitoring-config` config map in the `openshift-monitoring` namespace for core platform monitoring.
+* The `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace for components that monitor user-defined projects.
+
+.Prerequisites
+
+* *If you are configuring core platform monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` cluster role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
```suggestion
** You created a `ConfigMap` object named `cluster-monitoring-config`.
```
I tried to separate `ConfigMap` from `cluster-monitoring-config` so that the two monospace terms wouldn't be misinterpreted.

Reply:
Excellent suggestion. Changed.